frontend/sozisel-app/src/components/ParticipantActiveSession/ParticipantActiveSession.tsx

```diff
@@ -81,39 +53,99 @@ export default function ParticipantActiveSession({
     },
   });
 
-  return (
-    <>
-      <BasicNavbar />
-      <div className="ParticipantActiveSession">
-        <div className="jitsi">
-          {!loading && data?.generateJitsiToken.token && (
-            <JitsiFrame
-              roomId={id}
-              token={data.generateJitsiToken.token}
-              displayName={data.generateJitsiToken.displayName}
-            />
-          )}
+  useEffect(() => {
+    if (eventData?.eventLaunched) {
+      setActiveEvent(eventData.eventLaunched);
```

Comment:
And what happends when somebody enters the session after the event is launched? We probably need some query for most recent event.

Reply:
I think we shouldn't consider such case. When somebody enters the session after the event is launched he simply won't take part in that event.

Reply:
Ok, just add some comment that will suggest doing something like this in the future